pkg/plugins/platform/local_test.go

```diff
@@ -67,6 +68,15 @@ func TestMain(m *testing.M) {
 	os.Exit(testRun)
 }
 
+// TestLocalSetup reduces the TestSite list on shorter test runs.
+func TestLocalSetup(t *testing.T) {
+	// Allow tests to run in "short" mode, which will only test a single site. This keeps test runtimes low.
+	// We would much prefer to do this in TestMain, but the Short() flag is not yet available at that point.
+	if testing.Short() {
+		TestSites = []testcommon.TestSite{TestSites[0]}
```

Comment:
I guess you did this here because testing.T isn't available in TestMain, but I sure wish it was otherwise. I hate test dependencies and we already have a problem with this. TestLocalStart is doing required setup work, etc.  Is there any way to recover testing.T in TestMain for this kind of thing?